domains/test/service/src/domain.rs

```diff
@@ -317,13 +317,14 @@ where
     }
 
     /// Sends an system.remark extrinsic to the pool.
-    pub async fn send_remark_extrinsic(&mut self) -> Result<(), RpcTransactionError> {
+    pub async fn ensure_bundle_production(&mut self) {
```

Comment:
Why do we still need this, this can't really ensure bundle production, if the extrinsic is out of the operator's tx_range bundle will still be skipped.

Reply:
Yes there is no guarantee and maybe this needs to called competing else but I would still want to have someway to send a txn instead of hoping there is still an unconfirmed er while testing. I would rather keep it. If you have a better alternative name to it, I'm happy to take it

Reply:
Perhaps keep the name `send_remark_extrinsic` and have some comments and TODO there, we can revisit once we have a better idea.

Reply:
i remember you mentioning about not ok with the `send_remark_extrinsic ` so I changed it. Happy to change back :)

Reply:
I did mention this above, but it is not about its name, it is about whether we need it at all, sorry for the ambiguity.